Game.java

```diff
@@ -1,53 +1,98 @@
 import java.io.InputStream;
+import java.util.Random;
 import java.util.Scanner;
 
 public class Game {
 
+    private final Scanner scanner;
     private char turn = 'o';
     private InputStream stream;
+    private Board board;
+    private String gameMode = "pvp";
 
-    public Game(InputStream stream) {
+    public Game(InputStream stream, Board board) {
         this.stream = stream;
```

Comment:
you have an inputStream, what about an outputStream instead of the System.out's?